RecoTracker/TkHitPairs/src/CosmicHitPairGeneratorFromLayerPair.cc

```diff
@@ -16,20 +16,18 @@ CosmicHitPairGeneratorFromLayerPair::CosmicHitPairGeneratorFromLayerPair(
     const LayerWithHits* inner,
     const LayerWithHits* outer,
     //							     LayerCacheType* layerCache,
-    const edm::EventSetup& iSetup)
-    : TTRHbuilder(nullptr),
-      trackerGeometry(nullptr),
+    const TrackerGeometry& geom)
+    : trackerGeometry(&geom),
       //theLayerCache(*layerCache),
       theOuterLayer(outer),
       theInnerLayer(inner) {
-  edm::ESHandle<TrackerGeometry> tracker;
-  iSetup.get<TrackerDigiGeometryRecord>().get(tracker);
-  trackerGeometry = tracker.product();
+  //edm::ESHandle<TrackerGeometry> tracker;
+  //iSetup.get<TrackerDigiGeometryRecord>().get(tracker);
+  //trackerGeometry = tracker.product();
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant.
... here it looks like it can be removed

Reply:
removed